interpreter/src/main/java/org/intocps/maestro/interpreter/values/derivativeestimator/DerivativeEstimatorValue.java

```diff
@@ -19,13 +19,13 @@ public DerivativeEstimatorValue() {
             checkArgLength(fcargs, 3);
 
             List<Integer> indicesOfInterest =
-                    ValueExtractionUtilities.getArrayValue(fcargs.get(0), UnsignedIntegerValue.class).stream().map(UnsignedIntegerValue::intValue).collect(
+                    ValueExtractionUtilities.getArrayValue(fcargs.get(0), NumericValue.class).stream().map(NumericValue::intValue).collect(
```

Comment:
this should probably be long. Not sure if this is respected in all places

Reply:
Im not sure i follow why it should be long instead of integer?